lib/routes/sehuatang/index.ts

```diff
@@ -61,16 +62,24 @@ async function handler(ctx) {
     const type = ctx.req.param('type');
     const typefilter = type ? `&filter=typeid&typeid=${type}` : '';
     const link = `${host}forum.php?mod=forumdisplay&orderby=dateline&fid=${subformId}${typefilter}`;
-    const headers = {
-        'Accept-Encoding': 'gzip, deflate, br',
-        'Accept-Language': 'zh-CN,zh;q=0.9,en;q=0.8',
-        Cookie: '_safe=vqd37pjm4p5uodq339yzk6b7jdt6oich',
-    };
 
-    const response = await got(link, {
-        headers,
+    const browser = await puppeteer();
+    const page = await browser.newPage();
+    await page.setRequestInterception(true);
+    page.on('request', (request) => {
+        request.continue();
```

Comment:
Please do filter unnecessary requests when using puppeteer.
```suggestion
        request.resourceType() === 'document' || request.resourceType() === 'script' ? request.continue() : request.abort();
```